src/core/qgsmapsettings.cpp

```diff
@@ -436,6 +436,36 @@ QgsCoordinateTransform QgsMapSettings::layerTransform( const QgsMapLayer *layer
   return QgsCoordinateTransform( layer->crs(), mDestCRS, mTransformContext );
 }
 
+QgsRectangle QgsMapSettings::computeExtentForScale( const QgsPointXY &center, double scale ) const
+{
+  // Output width in inches
+  double outWIn = outputSize().width() / double( outputDpi() );
```

Comment:
```suggestion
  const double outputWidthInInches = outputSize().width() / double( outputDpi() );
```
(wordy naming is better than confusing names!)